pkg/config/config.go

```diff
@@ -210,6 +212,28 @@ func NewCreateChannelTx(channelConfig Channel, channelID string) (*cb.Envelope,
 	return env, nil
 }
 
+// NewSystemChannelBlock creates a genesis block using the provided consortiums and orderer
+// configuration and returns a block.
+func NewSystemChannelBlock(channelConfig Channel, channelID string) (*cb.Block, error) {
+	var err error
+
+	if channelID == "" {
+		return nil, errors.New("system channel ID is required")
+	}
+
+	ct, err := newSystemChannelGroup(channelConfig)
+	if err != nil {
+		return nil, fmt.Errorf("creating system channel group: %v", err)
+	}
+
+	block, err := newSystemChannelBlock(ct, channelID)
+	if err != nil {
+		return nil, fmt.Errorf("creating system channel block: %v", err)
```

Comment:
likewise to my comment on the function name, I think this error will need some rework too. We probably want to focus more on the fact that this is the genesis block as otherwise `creating system channel block` can be misconstrued for what this function is intended to be used for.

Reply:
I also added "genesis" in the error.